README.md

```diff
@@ -1,2 +1,42 @@
-# transportation-vz-data
-Project management and documentation for Vision Zero data management project in Knack
+# Vision Zero Editor (VZE)
+
+This project seeks to centralize and streamline the management of ATD's Vision Zero (VZ) data. As a result of this project, staff will have a standardized interface for reviewing crash data, prioritizing intersection safety improvements, and communicating efforts to the public. Additionally, high quality VZ data will be publicly accessible online.
+
+Data is sourced from TxDOT's Crash Records Information System (CRIS) database. An editing interface provides tools for enriching data with more attributes and correct erronous or missing data.
+
+For resources and updates, see the [Vision Zero Crash Data System](https://github.com/cityofaustin/atd-data-tech/issues/255) project index.
+
+## Getting Started
+
+### Technology Stack
+
+Technologies, libraries, and languages used for this project include:
+
+- Hasura
+- PostgreSQL
+- React (Javascript)
+- Core UI (HTML & CSS)
+- Capybara (Ruby)
+- Python
+
+### `atd-vze` Vision Zero Editor (VZE)
+
+Install dependencies
+
+`npm install`
+
+Run development server
+
+`npm start`
+
+In development, this project uses [Prettier](https://prettier.io/) for code formatting which is set in .prettierrc. Visit link for installation or install the [extension for VSCode](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode).
+
+### `atd-cris-capybara`
+
+TODO
```

Comment:
@sergiogcx, could you follow up on this task and add a bit about setup and getting Capybara running locally on ones machine?